src/test/java/test_cases/LoginTests.java

```diff
@@ -48,15 +47,27 @@ public void TC03() {
     String actualErrorMessage = loginPage.getErrorMessage();
     String expectedErrorMessage = "There was a problem with your login and/or errors exist in your form.";
 
-    Assert.assertEquals(actualErrorMessage, expectedErrorMessage, "Error message is not displayed");
+    Assert.assertEquals(actualErrorMessage, expectedErrorMessage, "Error message does not display");
   }
 
   @Test(description = "Login page displays when un-logged User clicks on Book ticket tab")
   public void TC04() {
     homePage.clickBookTicket();
+
     String actualPageHeader = homePage.getPageHeader();
     String expectedPageHeader = "Login Page";
 
-    Assert.assertEquals(actualPageHeader, expectedPageHeader, "Error Message is not displayed");
+    Assert.assertEquals(actualPageHeader, expectedPageHeader, "Login Page does not display");
+  }
+
+  @Test(description = "Additional pages display once user logged in")
+  public void TC06() {
+    User user = new User(Constant.USERNAME, Constant.PASSWORD);
+    homePage.clickLoginTab();
+    loginPage.login(user);
+
+    Assert.assertTrue(homePage.isMyTicketTabDisplayed(), "My ticket Tab does not display");
+    Assert.assertTrue(homePage.isLogOutTabDisplayed(), "Log out Tab does not display");
+    Assert.assertTrue(homePage.isChangePasswordTabDisplayed(), "Change Password does not display");
```

Comment:
Missing checkpoints:
click tab and verify page is displayed

Reply:
Yes, I added checkpoints click tab and verify page is displayed